src/modules/typography.css

```diff
@@ -29,14 +46,14 @@ p {
 }
 
 a {
-  color: var(--actionColor);
-  text-decoration: none;
+  color: var(--headingFontColor);
```

Comment:
I would make a new var for this --linkColor even if they have the same value, also maybe we can call the other one just --headingColor ?